File: 1074.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int n, x;
    cin >> n;
    while(n--){
        cin >> x;
        if(x==0){
            cout << "NULL\n";
        }else if (x%2==0){
            cout << "EVEN ";
            if(x>0){
                cout << "POSITIVE\n";
            }else{
                cout << "NEGATIVE\n";
            }
        }else{
            cout << "ODD ";
            if(x>0){
                cout << "POSITIVE\n";
            }else{
                cout << "NEGATIVE\n";
            }
        }
        
    }
}

File: 1074.py
a = int(input())

while(a>0):
    b = int(input())
    if b == 0:
        print("NULL")
    elif b%2 == 0:
        if b>0:
            print("EVEN POSITIVE")
        else:
            print("EVEN NEGATIVE")
    else:
        if b>0:
            print("ODD POSITIVE")
        else:
            print("ODD NEGATIVE")
    a-=1
